test/jasmine/bundle_tests/ie9_test.js

```diff
@@ -0,0 +1,40 @@
+var Plotly = require('@lib/core');
+
+Plotly.register([
+    require('@lib/bar'),
+    require('@lib/box'),
+    require('@lib/heatmap'),
+    require('@lib/histogram'),
+    require('@lib/histogram2d'),
+    require('@lib/histogram2dcontour'),
+    require('@lib/pie'),
+    require('@lib/contour'),
+    require('@lib/scatterternary'),
+    require('@lib/ohlc'),
+    require('@lib/candlestick')
+]);
+
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+
+describe('Bundle with IE9 supported trace types:', function() {
+
+    afterEach(destroyGraphDiv);
+
+    it('[wip] check that ie9_mock.js did its job', function() {
+        expect(window.ArrayBuffer).toBeUndefined();
+        expect(window.Uint8Array).toBeUndefined();
```

Comment:
cool - can we also do something like
```js
expect(function() { return ArrayBuffer; }).toThrow('ReferenceError: ArrayBuffer is not defined');
```
to make sure it's really inaccessible in the way it *actually* gets invoked in the code?